ui-admin/src/study/participants/participantList/ParticipantList.tsx

```diff
@@ -130,15 +129,11 @@ function ParticipantList({ studyEnvContext }: {studyEnvContext: StudyEnvContextT
 
   const searchEnrollees = async (portalShortcode: string, studyShortcode: string,
     envName: string, facetValues: FacetValue[]) => {
-    setIsLoading(true)
-    try {
+    doApiLoad(async () => {
       const response = await Api.searchEnrollees(portalShortcode, studyShortcode, envName,
         facetValues)
       setParticipantList(response)
-    } catch (e) {
-      Store.addNotification(failureNotification('Error loading participants'))
-    }
-    setIsLoading(false)
+    }, { setIsLoading })
   }
 
   useEffect(() => {
```

Comment:
This is what I wrote in an attempt to replace this `useEffect` (and the `searchEnrollees` function that only it uses) with `useLoadingEffect`:
```
  const { isLoading } = useLoadingEffect(async () => {
    const response = await Api.searchEnrollees(portal.shortcode, study.shortcode, currentEnv.environmentName,
      facetValues)
    setParticipantList(response)
  }, [portal.shortcode, study.shortcode, currentEnv.environmentName])
```
Does that look to you like it would work?